client/elements/addons/sc-top-sheet-views.js

```diff
@@ -330,6 +338,31 @@ export class SCTopSheetViews extends LitLocalized(LitElement) {
     super.connectedCallback();
     this._loadScToasts();
     this._fetchReferenceDisplayType();
+    document.addEventListener('keydown', this._handleKeydown.bind(this));
+    window.setTimeout(() => {
+      this.show();
+      this.hide();
+    }, 2000);
+  }
+
+  // This componenet is _never_ disconnected the way the code is written today (2024-08-04)
+  // Still, adding this in case some day this component is properly unloaded.
+  disconnectedCallback() {
```

Comment:
I was expecting this callback to be called when navigating from a sutta page to e.g. the homepage, but it wasn't called.  It seems that the top sheets are never unloaded?

Reply:
Yes, this component is not actively uninstalled from the DOM.